java-to-zod-gradle-plugin/src/test/java/sh/ivan/zod/resources/TestPersonClass.java

```diff
@@ -0,0 +1,89 @@
+package sh.ivan.zod.resources;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+
+import java.util.Date;
+import java.util.Objects;
+
+public final class TestPersonClass {
```

Comment:
Let's not add more tests here. I'd like to maintain exact test parity with the Maven plugin. This can be accomplished in a few steps:
1. Move the POJOs in [`java-to-zod-maven-plugin-test`](https://github.com/ivangreene/java-to-zod/tree/main/java-to-zod-maven-plugin-test/src/main/java/sh/ivan/pojo) into their own new submodule named `java-to-zod-plugin-test-support`
2. Bring that module in as a dependency to both `java-to-zod-maven-plugin-test` and to the module where we test  the Gradle plugin
3. Find a way to share the JS files (symlink a directory that contains them in both projects maybe?): https://github.com/ivangreene/java-to-zod/tree/main/java-to-zod-maven-plugin-test/js - not the schemas file though, that will get generated by each plugin
4. The test of the Gradle plugin should exist as a separate directory. It will be the only Gradle project in the tree. It will configure the Gradle plugin, and then run the same JS tests. I'm assuming that's mostly what this gradle build file does? https://github.com/ivangreene/java-to-zod/blob/6735d0785a0fb59b5e655b49d21997ebed93d6a4/java-to-zod-gradle-plugin/src/test/resources/build.gradle.kts